Verbatim `util/bigcat.v` section of patches/prosa-v06-rocq93-official-proof-closure-common.patch
(sha256 6819bfa9ac0bb9cfd6bfc7182d57e22463cf7a57b38ce9af89c8e7de96868050), split out for classic official closures.
--- a/util/bigcat.v
+++ b/util/bigcat.v
@@ -2,6 +2,7 @@
 Require Export prosa.util.tactics prosa.util.notation.
 From mathcomp Require Import ssreflect ssrbool eqtype ssrnat seq fintype bigop.
 Require Export prosa.util.tactics prosa.util.list.
+Set SsrOldRewriteGoalsOrder.
 
 (** In this section, we introduce lemmas about the concatenation
     operation performed over arbitrary sequences. *)
